jetbrains-core/src/software/aws/toolkits/jetbrains/services/lambda/execution/sam/SamRunningState.kt

```diff
@@ -95,4 +99,8 @@ class SamRunningState(
         eventFile.writeText(settings.input)
         return eventFile.absolutePath
     }
+
+    override fun createConsole(executor: Executor): ConsoleView? = runBlocking(getCoroutineUiContext(disposable = environment)) {
+        super.createConsole(executor)
```

Comment:
comment on the fact that certain consoles need to be created on EDT?
(though at this point we've probably switched between on and off EDT at least 3 times by now)

Reply:
Added